elements/pfelement/src/pfelement.js

```diff
@@ -475,7 +475,10 @@ class PFElement extends HTMLElement {
    */
   render() {
     this.shadowRoot.innerHTML = "";
-    this.template.innerHTML = this.html;
+    this.template.innerHTML = "";
+
+    if (this.styles) this.template.innerHTML += this.styles;
```

Comment:
Separating out styles and HTML makes it easier to put the template in the JS file but keep the stylesheet in Sass